### PR TITLE
Chore/update_webpack

### PR DESCRIPTION
**Pull request by MatiasOS**
This is an update of webpack to v4, also I have updated Storybook to v4.

The dist is the first approach of the build process. This will be updated and optimized later.

### CODE REVIEW

File: dist/styles.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
+/******/ (function(modules) { // webpackBootstrap
```
Comment (skkr) [inline dist/styles.css]: The CSS file was not generated correctly, and there is another styles.css in the dist/css folder.

File: webpack/prod/webpack.prod.config.js [hunk 2]
```diff
@@ -3,94 +3,36 @@ const path = require('path');
...
-        loader: 'babel-loader'
-      },
+    rules: [
       {
         test: /\.scss$/,
-        use: [
-          {loader: 'style-loader'},
-          {loader: 'css-loader'},
-          {
-            loader: 'postcss-loader',
```
Comment (skkr) [inline webpack/prod/webpack.prod.config.js]: We should keep the postcss loader and its config file.

File: webpack/prod/webpack.prod.config.js [hunk 3]
```diff
@@ -3,94 +3,36 @@ const path = require('path');
...
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
 
 const DIR_ROOT = path.join(__dirname, '../../');
 const DIR_SOURCE = path.resolve(DIR_ROOT, 'src') + '/';
-const DIR_PUBLIC = path.resolve(DIR_ROOT, 'public') + '/';
-
-const GIT_REV = process.env.GIT_REV || 'unknown';
-const VERSION_NUMBER = version || '0.0.0';
-
-console.log('Building App ... \n================\n');
```
Comment (skkr) [inline webpack/prod/webpack.prod.config.js]: Can we keep the building message? Perhaps "Building dist folder ..."

File: package.json [hunk 4]
```diff
@@ -60,9 +61,7 @@
     "cssnano": "^3.10.0",
```
Comment (skkr) [inline package.json]: If we are not going to use cssnano feel free to remove it.

File: dist/styles.css (hunk 1 above)
Comment (MatiasOS) [reply dist/styles.css]: done

File: webpack/prod/webpack.prod.config.js (hunk 2 above)
Comment (MatiasOS) [reply webpack/prod/webpack.prod.config.js]: I have added it back

File: webpack/prod/webpack.prod.config.js (hunk 3 above)
Comment (MatiasOS) [reply webpack/prod/webpack.prod.config.js]: Ok, It's back

File: package.json (hunk 4 above)
Comment (MatiasOS) [reply package.json]: removed

File: README.md [hunk 5]
```diff
@@ -1,25 +1,16 @@
...
+### Develop
+* Use `npm run storybook-dev` to start storybook in development mode.
 
+### Build
+#### Storybook
+* Use `npm run storybook-build` to build the static version of Storybook.
+* Use `npm run storybook-start` to run the static version of Storybook.
 
+#### npm
+* Use `npm runnpm-build` to compile `scss` files.
```
Comment (skkr) [inline README.md]: There is a space missing.

**Pull request merged**